guides/source/autoloading_and_reloading_constants.md

```diff
@@ -100,16 +100,16 @@ Also, engines can push in body of the engine class and in their own `config/envi
 
 WARNING. Please do not mutate `ActiveSupport::Dependencies.autoload_paths`; the public interface to change autoload paths is `config.autoload_paths`.
 
-WARNING: You cannot autoload code in the autoload paths while the application boots. It particular, directly in `config/initializers/*.rb`. Please check [_Autoloading when the application boots_](#autoloading-when-the-application-boots) down below for valid ways to do that.
+WARNING: You cannot autoload code in the autoload paths while the application boots. In particular, directly in `config/initializers/*.rb`. Please check [_Autoloading when the application boots_](#autoloading-when-the-application-boots) down below for valid ways to do that.
 
 The autoload paths are managed by the `Rails.autoloaders.main` autoloader.
 
 config.autoload_once_paths
 --------------------------
 
-You may want to be able to autoload classes and modules without reloading them. The autoload once paths store code that can be autoloaded, but won't be reloaded.
+It may be desired to autoload classes and modules without reloading them. The autoload once paths store code that can be autoloaded, but won't be reloaded.
```

Comment:
Thank you @p8, here's the clearer suggestion to your suggestion:

```suggestion
It may be desired to autoload classes and modules without reloading them. The `autoload_once_paths` configuration stores code that can be autoloaded, but won't be reloaded.
```

What do you think?